ingestion/src/metadata/ingestion/ometa/superset_rest.py

```diff
@@ -58,7 +58,13 @@ class SupersetAuthenticationProvider(AuthenticationProvider):
 
     def __init__(self, config: SupersetConfig):
         self.config = config
-        client_config = ClientConfig(base_url=config.url, api_version="api/v1")
+        client_config = ClientConfig(
+            base_url=config.url,
+            api_version="api/v1",
+            auth_token=lambda: ("no_token", 0),
```

Comment:
any reason we are adding no_token here @ulixius9

Reply:
because this client is used only to hit the login api.